File: 2988/2988.cpp
class Solution {
public:
    int managerOfTheLargestDepartment(TreeNode* root) {
        if(!root) return 0;
        return 1+max(managerOfTheLargestDepartment(root->left),managerOfTheLargestDepartment(root->right));
    }
};

File: 2988/2988.cs
public class Solution {
    public int ManagerOfTheLargestDepartment(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(ManagerOfTheLargestDepartment(root.left), ManagerOfTheLargestDepartment(root.right));
    }
}